xmodule/modulestore/tests/test_mixed_modulestore.py

```diff
@@ -724,6 +728,29 @@ def test_publish_automatically_after_delete_unit(self, default_ms):
         self.store.delete_item(vertical.location, self.user_id)
         assert not self._has_changes(sequential.location)
 
+    @ddt.data(ModuleStoreEnum.Type.mongo, ModuleStoreEnum.Type.split)
+    def test_course_create_event(self, default_ms):
+        """
+        Check that COURSE_CREATED event is sent when a course is created.
+        """
+        self.initdb(default_ms)
+        event_receiver = Mock()
+
+        COURSE_CREATED.connect(event_receiver)
+        test_course = self.store.create_course('test_org', 'test_course', 'test_run', self.user_id)
```

Comment:
Isn't `COURSE_CREATED.connect(event_receiver)` part of assignments?  @JuanDavidBuitrago 
```suggestion
        event_receiver = Mock()
        COURSE_CREATED.connect(event_receiver)

        test_course = self.store.create_course('test_org', 'test_course', 'test_run', self.user_id)
```